### PR TITLE
Issues/269 stubrunner grabdeps

### PR DESCRIPTION
**Pull request by marcingrzejszczak**
* now stub-runner uses the same configuration logic as stub-runner-spring so executing it is just the case of configuration
* Moved some classes around

### CODE REVIEW

**Comment by marcingrzejszczak** [comment]
Argh I forgot about build.gradle in stub-runner and providing a task to run main from gradle

File: stub-runner/stub-runner/src/main/groovy/com/ofg/stub/Arguments.groovy [hunk 1]
```diff
@@ -4,31 +4,30 @@ import com.ofg.stub.mapping.ProjectMetadata
...
+        this.repositoryPath = repositoryPath
         this.projects = projects
-        this.localZookeeperPath = localZookeeperPath
+        this.serviceName = serviceName
     }
 
-    Arguments(String repositoryPath, String projectRelativePath, Integer testingZookeeperPort, Integer minPortValue, Integer maxPortValue, String context, String localZookeeperPath) {
-        this(repositoryPath, projectRelativePath, testingZookeeperPort, minPortValue, maxPortValue, context, localZookeeperPath, null)
+    Arguments(StubRunnerOptions stubRunnerOptions, String context, String repositoryPath, String serviceName) {
+        this(stubRunnerOptions, context, repositoryPath, serviceName, null)
```
Comment (szpak) [inline stub-runner/stub-runner/src/main/groovy/com/ofg/stub/Arguments.groovy]: The second constructor could be replaced with default value `null` for `projects` parameter. Groovy compiler generates the same code.

File: stub-runner/stub-runner/src/main/groovy/com/ofg/stub/StubRunner.groovy [hunk 2]
```diff
@@ -1,70 +1,31 @@
...
-
-    @Option(name = "-maxp", aliases = ['--maxPort'], usage = "Maximum port value to be assigned to the Wiremock instance (e.g. 12345)", required = true)
-    private Integer maxPortValue
-
-    @Option(name = "-c", aliases = ['--context'], usage = "Context for which the project should be run (e.g. 'pl', 'lt')", required = true)
-    private String context
-
-    @Option(name = "-n", aliases = ['--serviceName'], usage = "Name of the service for which the project should be run (e.g. 'com/service/name')")
-    private String serviceName
+    private static final ThreadLocal<StubRunnerExecutor> STUB_RUNNER = new ThreadLocal<>()
```
Comment (szpak) [inline stub-runner/stub-runner/src/main/groovy/com/ofg/stub/StubRunner.groovy]: That is not a constant value, only constant reference to mutable object. I would keep the name `stubRunner`.

Comment (szpak) [inline stub-runner/stub-runner/src/main/groovy/com/ofg/stub/StubRunner.groovy]: Or even a name suggesting that is it a container for a stub runner (not stubRunner itself).

**Comment by szpak** [comment]
+1

File: stub-runner/stub-runner/src/main/groovy/com/ofg/stub/Arguments.groovy (hunk 1 above)
Comment (marcingrzejszczak) [inline stub-runner/stub-runner/src/main/groovy/com/ofg/stub/Arguments.groovy]: +1

File: stub-runner/stub-runner/src/main/groovy/com/ofg/stub/StubRunner.groovy (hunk 2 above)
Comment (marcingrzejszczak) [inline stub-runner/stub-runner/src/main/groovy/com/ofg/stub/StubRunner.groovy]: I named it stubRunnerContainer

**Comment by marcingrzejszczak** [comment]
Once the build passes I'll try to merge but don't hesitate to do that before me ;)

**Pull request merged**